website/docs/flow/index.md

```diff
@@ -1,80 +1,100 @@
 ---
-title: Overview
+title: Flows
 ---
 
-Flows are a method of describing a sequence of stages. A stage represents a single verification or logic step. They are used to authenticate users, enroll them, and more.
+Flows are a major component in authentik. In conjunction with stages and [policies](../policies/index.md), flows are at the heart of our system of building blocks, used to define and execute the workflows of authentication.
```

Comment:
Would probably add things "authorization", "enrollment" and "user settings" to the end of this